src/Qir/Tests/Tools/TestCases/FullStateDriverGenerator/UsePauliArg.cpp

```diff
@@ -43,19 +32,9 @@ int main(int argc, char* argv[])
     CLI::App app("QIR Standalone Entry Point");
 
     // Initialize runtime.
-    unique_ptr<IRuntimeDriver> sim = CreateFullstateSimulator();
-    QirContextScope qirctx(sim.get(), false /*trackAllocatedObjects*/);
-
-    // Add the --simulation-output option.
-    string simulationOutputFile;
-    CLI::Option* simulationOutputFileOpt = app.add_option(
-        "--simulation-output",
-        simulationOutputFile,
-        "File where the output produced during the simulation is written");
-
     // Add a command line option for each entry-point parameter.
-    PauliId PauliArgCli;
-    PauliArgCli = PauliId::PauliId_I;
+    char PauliArgCli;
+    PauliArgCli = 0;
```

Comment:
The Pauli values turn into the "magic numbers". E.g. it stops being evident that in green L37 and L20 the same value is used.
Consider creating some common C header with Pauli values enum, other common stuff, and including that header wherever needed.